tcat-app/app.py

```diff
@@ -657,8 +669,77 @@ def download_run_data(scenario_id, config_run_id):
+                      data['incObjectDisp'] * astro_units.deg if data['incObjectDisp'] is not None else None,
+                      data['adrStage'],
+                      data['mAdr'] * astro_units.kg if data['mAdr'] is not None else None,
+                      data['adrCrossSection'] * astro_units.m ** 2 if data['adrCrossSection'] is not None else None,
+                      data['adrMeanThrust'] * astro_units.N if data['adrMeanThrust'] is not None else None,
+                      data['adrIsp'] * astro_units.s if data['adrIsp'] is not None else None,
+                      data['adrManoeuvreSuccess'],
+                      data['adrCaptureSuccess'],
+                      data['mDebris'] * astro_units.kg if data['mDebris'] is not None else None,
+                      data['debrisCrossSection'] * astro_units.m ** 2 if data['debrisCrossSection'] is not None else None,
+                      data['apogeeDebris'] * astro_units.km if data['apogeeDebris'] is not None else None,
+                      data['perigeeDebris'] * astro_units.km if data['perigeeDebris'] is not None else None,
+                      data['incDebris'] * astro_units.deg if data['incDebris'] is not None else None,
+                      data['apogeeDebrisRemoval'] * astro_units.km if data['apogeeDebrisRemoval'] is not None else None,
+                      data['perigeeDebrisRemoval'] * astro_units.km if data['perigeeDebrisRemoval'] is not None else None,
+                      data['incDebrisRemoval'] * astro_units.deg if data['incDebrisRemoval'] is not None else None,
+                      os.path.join(TCAT_DIR, 'ACT_Space_Debris_Index/sdi_space_debris_CF_for_code.csv'),
+                      os.path.join(TCAT_DIR, 'ACT_Space_Debris_Index/sdi_reduced_lifetime.csv'))
+
+    response = {'LCS3': result['LCS3'].value, 'LCS4': result['LCS4'].value}
+    return jsonify(response)
+
+
+@app.route('/api/v1/calculations/atm', methods=['POST'])
+@oidc.accept_token(require_token=True)
+def get_atm():
+    data = request.get_json()
+
+    if data is None:
+        return '{"error": "No data provided"}'
+
+    result = atm_main(TCAT_DIR,
+                      data['launcher'],
+                      data['engine'],
+                      data['numberOfEngines'],
+                      data['propType'],
+                      data['isp'] * astro_units.s if data['isp'] is not None else None,
+                      data['ignitionTimestamp'] * astro_units.s if data['ignitionTimestamp'] is not None else None,
+                      data['cutoffTimestamp'] * astro_units.s if data['cutoffTimestamp'] is not None else None,
+                      data['numberOfLaunches'],
+                      data['rawTrajectory'],
+                      data['rawThrustCurve'],
+                      plotting=False)
+
+    return jsonify(result)
+
+
 app.jinja_env.globals.update(inputparams=inputparams)
 
 if __name__ == '__main__':
-    app.run()
+    app.run(port=5555, debug=True)
```

Comment:
will you have to leave the debug = True ? What does it mean for the script ?

Reply:
Thanks for pointing this out. I added this locally for debugging. We can remove this.